src/controllers/dlgprefcontroller.cpp

```diff
@@ -749,3 +750,11 @@ void DlgPrefController::openScript() {
         }
     }
 }
+
+
+bool DlgPrefController::presetInfoLessThan(const PresetInfo &a, const PresetInfo &b) {
```

Comment:
Ok, thanks, I do it, I didn't know its a good programming practice to do like it, thanks I do it